docs/en_US/NNICTLDOC.md

```diff
@@ -388,6 +370,92 @@ Debug mode will disable version check function in Trialkeeper.
     nnictl experiment list
     ```
 
+<a name="export"></a>
+
+* __nnictl experiment export__
+  * Description
+
+    You can use this command to export reward & hyper-parameter of trial jobs to a csv file.
+
+  * Usage
+
+    ```bash
+    nnictl experiment export [OPTIONS]
+    ```
+
+  * Options
+
+  |Name, shorthand|Required|Default|Description|
+  |------|------|------ |------|
+  |id|  False| |ID of the experiment    |
+  |--file|  True| |File path of the output file     |
+  |--type|  True| |Type of output file, only support "csv" and "json"|
+
+  * Examples
+
+  > export all trial data in an experiment as json format
+
+  ```bash
+  nnictl experiment export [experiment_id] --file [file_path] --type json
```

Comment:
nnictl already have command ```nnictl trial export --file <fn>```,  it export trial results,  this command looks like can export entire experiment, but it can not.  Can we reconsider the command name?

Reply:
We have a meeting on Thursday morning to discuss changes.
Our plan is to change "nnictl trial export" to "nnictl experiment export", add "nnictl experiment import". It's for the user to use "experiment export" to download data and use "experiment import" to feed the data to another experiment. So the new design wants to put them together. The nnictl code logic modified in @Crysple pr: #pr980
Besides, we consider that "nnictl trial export" actually export all trials' info of this experiment, so it's acceptable for us to use "nnictl experiment".